to-do-app/src/AddTodo.js

```diff
@@ -0,0 +1,32 @@
+import React, { Component } from 'react'
+
+class AddTodo extends Component {
+    state = {
+        content: ''
+    }
+    handleChange = (e) => {
+        this.setState({
+            content: e.target.value
+        })
+    }
+    handleSubmit = (e) => {
+        e.preventDefault();
```

Comment:
Implement checking for empty `content` before submit. You shouldn't add empty items to the list.

Reply:
Add checking for empty content before submit